contracts/dynamic-callee-contract/src/contract.rs

```diff
@@ -37,13 +37,14 @@ pub struct ExampleStruct {
 }
 
 #[derive(Contract)]
-struct Caller {
+struct Callee {
```

Comment:
Why did you change from Caller to Callee?

Reply:
I thought it represents "Callee" and "Caller" is a mistake.